src/python/py/models/builder.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
         self.filename = extra_options["filename"] if "filename" in extra_options else "model.onnx"
         self.hf_token = parse_hf_token(extra_options.get("hf_token", "true"))
         self.extra_options = extra_options
+        self.quant_provider = extra_options["quant_provider"] if "quant_provider" in extra_options else "default"
+        self.nvidia_awq_calibration = extra_options["nvidia_awq_calibration"] if "nvidia_awq_calibration" in extra_options else "awq"
```

Comment:
Since other AWQ tools have a calibration option, can we generalize this name?

```suggestion
        self.calibration_method = extra_options["calibration_method"] if "calibration_method" in extra_options else "awq"
```

Reply:
This might cause problem if some other quant_provider get added future that supports some algo other then awq. In that case we might have different calibration methods